src/stories/Checkbox.js

```diff
@@ -0,0 +1,14 @@
+import { html } from "lit-html";
+
+/**
+ * Primary UI component for user interaction
+ */
+export const Checkbox = ({ disabled, label, checked, isError }) => {
+  return html`
+    <label role="checkbox" aria-invalid="${isError}">
+        <input type="checkbox" .disabled="${disabled}" .checked="${checked}"></input>
+        <span>${label}</span>
+    </label>
+    <span>¡Ouch! Este checkbox no esta tildado.</span>
```

Comment:
Creo que estaría bueno parametrizar el mensaje de error cómo ya hacés con el label:

```suggestion
    <span>${errorMessage}</span>
```

Reply:
Ojo! el parámetro es solo para storybook y en la vista uno pueda escribir cualquier cosa como label, no es propio del componente en si.

Reply:
Claro, pero ya que se parametriza el label, por que no el mensaje también?

Reply:
Podríamos colocar el mensaje **_"¡Ouch! Este checkbox no esta tildado."_** por default y luego si se quiere que lo pueda cambiar, creo que a eso te refieres @andresmoschini

Reply:
![isso-isso-isso-el-chavo-del-ocho](https://user-images.githubusercontent.com/1157864/221172716-2b2c6920-7e16-4a3c-afbb-8d4a1e9a473b.gif)